packages/api-db-mongodb/src/migration.ts

```diff
@@ -593,6 +593,26 @@ export const Migrations: Migration[] = [
         }
       )
     }
+  },
+  {
+    //  Rename street field in address to address
```

Comment:
This migration comment should probably be changed

Reply:
done. Thank you :)